canvas_sdk/effects/send_invite.py

```diff
@@ -0,0 +1,43 @@
+from typing import Any
+
+from pydantic_core import InitErrorDetails
+
+from canvas_sdk.effects.base import EffectType, _BaseEffect
+from canvas_sdk.v1.data import CanvasUser
+
+
+class SendInviteEffect(_BaseEffect):
+    """
+    An Effect that will send an invitation for the Patient Portal.
+    """
+
+    class Meta:
+        effect_type = EffectType.PATIENT_PORTAL__SEND_INVITE
+
+    user_dbid: int
+
+    @property
+    def values(self) -> dict[str, Any]:
+        """The user's id."""
+        return {"dbid": self.user_dbid}
```

Comment:
Missed this one sorry. Let's be explicit about what we're sending
```suggestion
        return {"user_dbid": self.user_dbid}
```